src/components/IconButton/IconButton.tsx

```diff
@@ -42,7 +42,14 @@ const IconButton = ({
     <Button
       {...commonProps}
       variant={variant}
-      className={classnames(baseClass, `${baseClass}--${variant}`, className)}
+      className={classnames(
+        baseClass,
+        `${baseClass}--${variant}`,
+        {
+          [`${baseClass}--is-disabled`]: isDisabled,
```

Comment:
I see you added styles for the Pagination version, but are there styles for this `--is-disabled` class for IconButton?

Reply:
ah yes this is not needed anymore